Neo4j.Driver/Neo4j.Driver/Neo4j.Driver.csproj

```diff
@@ -39,21 +39,21 @@
         <WarningLevel>2</WarningLevel>
     </PropertyGroup>
     <ItemGroup Condition="'$(TargetFramework)' == 'net5.0'">
-        <PackageReference Include="System.Runtime.InteropServices.RuntimeInformation" Version="4.3.0"/>
```

Comment:
for whatever reason our configs are not aligned on what this xml should have, we need to pick a style.